taxi/urls.py

```diff
@@ -1,5 +1,9 @@
+from mailcap import lookup
```

Comment:
The import `from mailcap import lookup` is unnecessary and can be removed unless it is used elsewhere in the file or project.